surmise/calibration.py

```diff
@@ -381,6 +420,41 @@ def lpdf(self, y=None, args=None):
         """
         raise ValueError('lpdf functionality not in method')
 
+    def save_to(self, filename):
+        """
+        Simple serialization and save function for calibrator prediction object.
+
+        :Example:
+
+        .. code-block:: python
+
+            cal = calibrator(...)
+
+            calpred = cal.predict(...)
+
+            calpred.save_to('calpred_example.pkl')
+
+            loaded_calpred = calibrator.load_from('calpred_example.pkl')

```

Comment:
From a design point of view, it is a bit unexpected to ask a prediction object to write itself to file, but use the calibration class to load the prediction.  Put `load_from` in prediction as well?

This makes some sense because it could allow each `load_from` method to, for instance, check the loaded data to confirm that it corresponds to the correct class.

Same for emulator.

Reply:
On a second thought, I may remove the prediction save/loads, because they can be accessible through an emulator/calibrator object.  In our context, predictions should take way less time than training.

Reply:
The original thought is that: prediction can only be called after having an emulator or calibrator object, so it doesn't make sense to load just from a prediction object.

Reply:
If you wanted to load predictions from the calibrator class, then I understand that you are somehow "hiding" the prediction class or making its existence less important.  If this is the goal and you want the ability to load just the prediction, then it might be nice to have a `calibrator.load_from` and `calibrator.load_prediction`.

This interface is clearer in the sense that I know what I am loading and what I get back.  It would also allow for the error checking that I mentioned and communicates your possible goal that predictions are accessed through calibrators.  However, even this might be somewhat strange because you wouldn't be getting the prediction from the actual calibrator object that created it, but rather from a static method.

Stripping down the interface to just saving/loading the calibrators does make sense if the compute is quick.  Some possible issues with this:
* Saving the prediction to file might be useful for storing a record of a research study.  If you store the calibrator are you guaranteed the ability to load it and reaccess the predictions a year or two later?  Would you have to use a newer version of the software that doesn't work with that pickle anymore or that produces results that are different from those that you used to publish in an important way (i.e., you can't reproduce your predictions due to changes in the code/algorithm)?
* What is the size of the calibrator object vs. the size of the prediction object?  If you only allow saving the calibrator and the only object that someone wants is the prediction, are you forcing them to store and manage a much larger file?

Reply:
Addressing in #136